README.md

```diff
@@ -1,26 +1,26 @@
-# Rust Model Checker (RMC)
-The Rust Model Checker (RMC) aims to be a bit-precise model-checker for Rust.
+# Rust Model Checker (Kani)
```

Comment:
How about the title: Kani Rust Verifier (Kani)?

Reply:
Yeah, that's a good point!